core/globals.h

```diff
@@ -308,6 +308,17 @@ typedef struct _dr_stats_t {
     uint64 synchs_not_at_safe_spot;
 } dr_stats_t;
 
+/**
+ * Error codes of DR API routines.
+ */
+typedef enum {
+    DR_ERROR_NONE,
+    DR_ERROR_INVALID_PARAM,
+    DR_ERROR_INSUFFICIENT_SPACE,
+    DR_ERROR_APP_ENCODING,
```

Comment:
Hard to know what this means from the name (see above: should have doc).  Maybe it should be DR_ERROR_UNKNOWN_ENCODING?

Reply:
Done.